crates/fullnode/src/runner.rs

```diff
@@ -485,7 +394,7 @@ where
                 sequencer_commitment.clone(),
             )?;
 
-            for i in start_l1_height..=end_l1_height {
+            for i in start_l2_height..=end_l2_height {
                 self.ledger_db.put_soft_confirmation_status(
                     SlotNumber(i),
```

Comment:
SlotNumber stands for L1, BatchNumber for L2

Reply:
good catch, types of that table should change.

back when sequencer commitments were for L1 range, it made sense to have the "L2 status" table's keys as L1 heights. (either all L2 blocks of an L1 block get committetd / proven or none) right now any range of L2 blocks can get committed / proven.

Reply:
I thought the table would be sufficient since `SlotNumber` can also hold L2 ranges. So our problem with this line is mostly aesthetic. I will introduce a new table since you guys agree i should.

Reply:
Done